README.md

```diff
@@ -1,2 +1,170 @@
-# rocket-mission-list
-In project, we will be working with the real live data from the SpaceX API. We go to build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Rocket Mission List</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Rocket-Mission-List] <a name="about-project"></a>
+
+**[Rocket-Mission-list]** is a React project, with the real live data from the SpaceX API.  We are going to  build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+React JS, Redux, JavaScript, HTML and CSS
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[UI to display rockets and booking/cancel a rocket]**
+- **[UI to display missions and join/leave a mission]**
+- **[Filter rockets reserved and missions joined and display them in MyProfile page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link] Not-yet-available
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to have installed Node JS
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:OsvaldoBC/rocket-mission-list.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd rocket-mission-list
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
```

Comment:
- [ OPTIONAL ] I recommend to add how to run the tests in the README since if anyone want to run your tests so he can run them for me I know but maybe another person, they don't know how! 💯🔥🚀 so please add this section here as well 🙏🏻😉✨